rasp/base.py

```diff
@@ -37,14 +37,37 @@ def get_page_source(self, url, data=None):
 
 class Webpage(object):
     def __init__(self, url=None, source=None):
-        self.url = url
-        self.source = source
+        """The Webpage object represents all we know about a single scraped page.
+
+        The Webpage object is the key object constructed by an engine to represent what we know about a given webpage.
+        It includes things like the page source, url, and date of access.
+
+        :param url: the url of the webpage you are representing
+        :param source: the source, as text of the webpage.
+        :return: Webpage
+        """
+
+        self._url = url
+        self._source = source
+        self._access_date = time.time()
+
+    @property
```

Comment:
Right now, there aren't any behaviors, but as we continue on, I think we will see things like character encodings, subsets of the values, etc. being useful to parse out in getter/setters.  One kind of trivial example would be to set some variable indicating whether it was accessed via https in the setter for the URL. 

So right now, no, but I think later on yes.